packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelectMenuItemsWithSearch.tsx

```diff
@@ -49,21 +52,15 @@ export const SingleEntitySelectMenuItemsWithSearch = ({
       excludedRelationRecordIds,
     });
 
-  const showCreateButton = isDefined(onCreate);
-
-  let onCreateWithInput = undefined;
-
-  if (isDefined(onCreate)) {
-    onCreateWithInput = () => {
-      if (onCreate.length > 0) {
-        (onCreate as (searchInput?: string) => void)(
-          relationPickerSearchFilter,
-        );
-      } else {
-        (onCreate as () => void)();
-      }
-    };
-  }
+  const createNewButton = isDefined(onCreate) && (
+    <DropdownMenuItemsContainer>
+      <CreateNewButton
+        onClick={() => onCreate?.(relationPickerSearchFilter)}
+        LeftIcon={IconPlus}
+        text="Add New"
+      />
+    </DropdownMenuItemsContainer>
```

Comment:
Indeed, as per @FelixMalfait's [comment](https://github.com/nicolasrouanne/twenty/pull/1#discussion_r1838662324), wrapping the "Add New" button inside a `<DropdownMenuItemsContainer>` does add some nice padding 👍🏻 

### Before
![Screenshot 2024-11-13 at 12 06 59](https://github.com/user-attachments/assets/cf7d20d2-fb2d-4754-a621-281159b9f460)

### After
![Screenshot 2024-11-13 at 12 06 49](https://github.com/user-attachments/assets/d83f23d8-da27-40f0-bb9f-0cce6fbccd9b)